tensorflow/compiler/mlir/tools/kernel_gen/tests/tf_64bit_indexer.mlir

```diff
@@ -0,0 +1,30 @@
+// RUN: kernel-gen-opt %s --split-input-file \
+// RUN:   --tf-64bit-indexer="architectures=sm_123,sm_456 \
```

Comment:
I think this test needs an update as `tf-64bit-indexer` was merged into `tf-to-jit-invocation`. 
Can we also rename the file accordingly?